packages/gatsby-plugin-create-client-paths/src/gatsby-node.js

```diff
@@ -18,7 +18,7 @@ exports.onCreatePage = ({ page, store, actions }, { prefixes }) => {
       return resolve()
     }
 
-    prefixes.some(prefix => {
+    prefixes.forEach(prefix => {
```

Comment:
github commmenting is dumb 💤 

can you also remove the return true & return false from this function as forEach doesn't care.

I'm also thinking if we should drop the promise. What use has it?

Reply:
Promise shouldn't be necessary since `createPage` is an action creator and therefore not async